x-pack/plugins/apm/public/components/app/ServiceMap/Popover/Contents.tsx

```diff
@@ -70,7 +71,8 @@ export function Contents({
       </FlexColumnItem>
       <FlexColumnItem>
         {isService ? (
-          <ServiceMetricFetcher
+          <ServiceStatsFetcher
+            environment={selectedNodeData[SERVICE_ENVIRONMENT]}
```

Comment:
I don't think we need to provide this as a prop since we're getting it from the UI filters.